gunicorn/config.py

```diff
@@ -1171,14 +1183,21 @@ class Group(Setting):
     cli = ["-g", "--group"]
     meta = "GROUP"
     validator = validate_group
-    default = os.getegid()
```

Comment:
again what does this change fix ? what is the error caused by this? We need to ensure the worker process run under the specified or running user.